blocks/library/verse/index.js

```diff
@@ -58,6 +58,7 @@ registerBlockType( 'core/verse', {
 				onFocus={ setFocus }
 				placeholder={ __( 'Write…' ) }
 				className={ className }
+				formattingControls={ [ 'bold', 'italic', 'strikethrough' ] }
```

Comment:
I was talking with @folletto about offering an author/source field instead. I think link detracts from the intention of the block a bit.